frontend/src/editor/automerge_websocket_editor.ts

```diff
@@ -62,17 +62,31 @@ export function useAutomergeWebsocketEditor(
           editor.setDoc(newDoc);
         }
       } else if (msg_type === MessageSyncType.ChangeBacklogComplete) {
+        console.info('backlog complete');
```

Comment:
I feel like this "batch load + migrate" should be factored out into a function or lambda and be used by the `ChangeBacklogComplete` and the `FullDoc` path